UnitTesting/UnitTesting.csproj

```diff
@@ -8,14 +8,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>UnitTesting</RootNamespace>
     <AssemblyName>UnitTesting</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Ah yeah this version needs to be changed back to 4.5.2 You can download the targetting pack for it from Microsoft. 4.5 support ended in January of this year.